### PR TITLE
[record-minmax] Derive moving average of minmax

### PR DESCRIPTION
**Pull request by jinevening**
This derives moving average of minmax

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to #1605

### CODE REVIEW

**Comment by jinevening** [comment]
This commit changes the way to derive the min/max values saved in QuantizationParameters. Now, we use weighted moving average instead of clipping + averaging.

File: compiler/record-minmax/src/RecordMinMax.cpp [hunk 1]
```diff
@@ -37,37 +37,20 @@ namespace
...
-  // Sort
-  std::sort(vector.begin(), vector.end());
-
-  // Clip and Average
-  double res = std::accumulate(vector.begin() + clip_bottom, vector.end() - clip_top, 0.0) /
-               (vector.size() - clipped_items);
-
-  return static_cast<float>(res);
+  float curr_avg = vector[0];
+  for (int i = 1; i < vector.size(); i++)
```
Comment (seanshpark) [inline compiler/record-minmax/src/RecordMinMax.cpp:49]: can you please use same type of `vector.size()` instead of `int` ?

**Comment by seanshpark** [comment]
others looks good

File: compiler/record-minmax/src/RecordMinMax.cpp (hunk 1 above)
Comment (jinevening) [reply compiler/record-minmax/src/RecordMinMax.cpp]: I've changed the type. Thanks.

File: compiler/record-minmax/src/RecordMinMax.cpp [hunk 2]
```diff
@@ -37,37 +37,20 @@ namespace
...
- * @brief  clipAndAverage does the following things
- *         (1) Discard the top/bottom clip_ratio (%) values from vector
- *             Ex: top_clip_ratio = 5, bottom_clip_ratio = 3
- *                 discard top 5 % values and bottom 3 % values from vector
- *         (2) Calculate the average of remaining values
+ * @brief  getMovingAverage calculates the weighted moving average of input vector
+ *         The initial value is the first element of the vector
  */
-float clipAndAverage(std::vector<float> &vector, float top_clip_ratio, float bottom_clip_ratio)
+float getMovingAverage(std::vector<float> &vector, float alpha = 0.9)
```
Comment (seanshpark) [inline compiler/record-minmax/src/RecordMinMax.cpp:43]: We usually didn't use giving default values like `float alpha = 0.9`. Can you remove `= 0.9` and set this to callers ?

**Pull request merged**